boards/arm/stm32/viewtool-stm32f107/configs/tcpblaster/defconfig

```diff
@@ -24,6 +24,7 @@ CONFIG_INTELHEX_BINARY=y
 CONFIG_IOB_NBUFFERS=50
 CONFIG_IOB_NCHAINS=12
 CONFIG_LIBC_HOSTNAME="Viewtool-STM32F107"
+CONFIG_LIBM=y
```

Comment:
Are the changes in `boards/arm/stm32/viewtool-stm32f107/configs/tcpblaster/defconfig` meant to be included in this PR?

Reply:
Yes, This PR is to resolve the break of ci, since clang-17 does not provide the implementation of libm, so in order to completely solve similar issues, it is necessary to enable the implementation of libm in some boards.